classroom/snippets/snippets.py

```diff
@@ -13,13 +13,110 @@
 # limitations under the License.
 
 from __future__ import print_function
+import json
 from googleapiclient import errors
 
 
 class ClassroomSnippets(object):
     def __init__(self, service):
         self.service = service
 
+    def create_course(self):
+        """
+        Creates a 10th Grade Biology course.
+        """
+        service = self.service
+        # [START classroom_create_course]
+        course = {
+            'name': '10th Grade Biology',
+            'section': 'Period 2',
+            'descriptionHeading': 'Welcome to 10th Grade Biology',
+            'description': """We'll be learning about about the
+                         structure of living creatures from a
+                         combination of textbooks, guest lectures,
+                         and lab work. Expect to be excited!""",
+            'room': '301',
+            'ownerId': 'me',
+            'courseState': 'PROVISIONED'
+        }
+        course = service.courses().create(body=course).execute()
+        print('Course created:', course.get('name'), course.get('id'))
+        # [END classroom_create_course]
+
+    def get_course(self):
+        """
+        Retrieves a classroom course by its id.
+        """
+        service = self.service
+        # [START classroom_get_course]
+        course_id = '123456'
+        try:
```

Comment:
Do we need a try/catch here? I'd model these after slides and return the response:
https://github.com/gsuitedevs/python-samples/blob/master/slides/snippets/slides_snippets.py#L132

Reply:
the other sample snippets also have try/catches -- should those eventually be modified too? i think it was included since the id could not exist as a course? https://developers.google.com/classroom/guides/manage-courses#retrieve_course_details

Reply:
Interesting -- unless this is a Python style preference I don't know about I'd prefer to keep them clean like this equivalent Slides snippets:
https://developers.google.com/slides/how-tos/presentations